src/pages/EditRequestPage.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import lodashGet from 'lodash/get';
+import {withOnyx} from 'react-native-onyx';
+import CONST from '../CONST';
+import ScreenWrapper from '../components/ScreenWrapper';
+import HeaderWithBackButton from '../components/HeaderWithBackButton';
+import Navigation from '../libs/Navigation/Navigation';
+import compose from '../libs/compose';
+import withLocalize, {withLocalizePropTypes} from '../components/withLocalize';
+import MoneyRequestAmountPage from './iou/steps/MoneyRequestAmountPage';
+import * as CurrencyUtils from '../libs/CurrencyUtils';
+import ONYXKEYS from '../ONYXKEYS';
+import * as ReportActionsUtils from '../libs/ReportActionsUtils';
+import RequestDescription from '../components/RequestDescription';
+import RequestCreated from '../components/RequestCreated';
+import DateUtils from '../libs/DateUtils';
+import reportPropTypes from './reportPropTypes';
+import * as ReportUtils from '../libs/ReportUtils';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+
+    /** Route from navigation */
+    route: PropTypes.shape({
+        /** Params from the route */
+        params: PropTypes.shape({
+            /** Which field we are editing */
+            field: PropTypes.string,
+
+            /** This is going to be the reportID for the "transaction thread" */
+            threadReportID: PropTypes.string,
```

Comment:
Could you explain what you mean by "going to be"

Reply:
its just bad english for "this is"